cea/inputlocator.py

```diff
@@ -468,6 +474,33 @@ def get_edge_mass_flow_csv_file(self, network_type):
         return os.path.join(self.get_optimization_network_layout_folder(), 'NominalEdgeMassFlow_' +
                              network_type + '.csv')
 
+    def get_network_layout_edges_shapefile(self, network):
```

Comment:
The folder name can be changed to `thermal_networks`
However, the shape files for nodes and edges have to be shipped with multiple dependent files so that geopandas can import all information from the attribute tables. It will be difficult to organize without having sub-folders.